src/childHealth/utils.py

```
@@ -0,0 +1,44 @@
+import pandas as pd
```

Ruff

`pandas` imported but unused

Remove unused import: `pandas`

(F401)